### PR TITLE
quick fix to make Pages work for prod Let's Encrypt

### PR DESCRIPTION
**Pull request by songgao**

### CODE REVIEW

File: libpages/server.go [hunk 1]
```diff
@@ -298,8 +298,8 @@ func ListenAndServe(ctx context.Context,
 		HostPolicy: server.allowConnectionTo,
 	}
 
-	if config.UseDiskCacheForDev {
-		manager.Cache = autocert.DirCache("./kbp-cert-cache-dev")
+	if config.UseDiskCertCache {
+		manager.Cache = autocert.DirCache("./kbp-cert-cache")
```
Comment (strib) [inline libpages/server.go]: Should this use a different directory if `!fProd`, so the certs don't get mixed up if you switch between modes on the same instance?

Comment (songgao) [reply libpages/server.go]: I figured we'd almost never use the same instance for testing and prod, so I just removed the `-dev` suffix. But having different suffix sounds good with me too; will change!

File: kbpagesd/main.go [hunk 2]
```diff
@@ -23,7 +23,7 @@ var (
 
 func init() {
 	flag.BoolVar(&fProd, "prod", false, "disable development mode")
```
Comment (jzila) [inline kbpagesd/main.go]: Should this be true now by default?

Comment (songgao) [reply kbpagesd/main.go]: I'd prefer having to explicitly set it. Same concept as requiring to set `KEYBASE_RUN_MODE` for non-dev environments. WDYT?

Comment (jzila) [reply kbpagesd/main.go]: Sounds good.

**Pull request merged**